### ISSUE TITLE
Title does not work

### ISSUE THREAD

**Issue by ixnoahlive** [sndyx/hsl#3]
`title` causes following error:
```rs
fn intro() {
    title("My title", "My subtitle", 1, 1, 1)
}
```

![image](https://github.com/sndyx/hsl/assets/70314622/16a446ff-da6d-444e-8430-9189b8d63f01)

**Comment by sndyx** [comment]
Yes, this is a bug which is fixed by adding a single string to a list. Will be fixed in the next release and will try to push this out ASAP.

**Comment by sndyx** [comment]
Fixed in [e017f30e](https://github.com/sndyx/hsl/commit/e017f30ec55dbc4dcfa9e7c013c4db1b9e48881b).
